cartography/data/jobs/cleanup/gcp_sql_cleanup.json

```diff
@@ -1,11 +1,23 @@
 {
 	"statements": [
 		{
-			"query": "MATCH (u:GCPSQLUser)<-[:USED_BY]-(:GCPSQLInstance)<-[:RESOURCE]-(:GCPProject{id: $GCP_PROJECT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE u.lastupdated <> $UPDATE_TAG WITH u LIMIT $LIMIT_SIZE DETACH DELETE (u) return COUNT(*) as TotalCompleted",
+			"query": "MATCH (u:GCPSQLUser)<-[:DATABASE]-(:GCPSQLInstance)<-[:RESOURCE]-(:GCPProject{id: $GCP_PROJECT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE u.lastupdated <> $UPDATE_TAG WITH u LIMIT $LIMIT_SIZE DETACH DELETE (u) return COUNT(*) as TotalCompleted",
```

Comment:
Why is the relationship `DATABASE` of `instance` with `user`

Reply:
by mistake change . revert back  now